e2e/cypress/integration/client-admin/home.spec.js

```diff
@@ -0,0 +1,12 @@
+describe('Home Page', () => {
+  beforeEach(() => cy.visit(''))
+
+  it('bare Url redirects to /home', () => {
+    cy.url().should('eq', Cypress.config().baseUrl + '/home')
+  })
+
+  it('has Sign up and Sign in links', () => {
+    cy.get('#root').find('a[href="/demo"]')
```

Comment:
This is not the "sign up" link, ie. "/createuser". Did that link go away?
Maybe just change the test title to 'has Demo and Sign in links'

Reply:
I was just making sure this succeeded on current `dev` build, but fully expect to revert back to what you had after the `client-admin` changes land there :)

Will change test name tho